src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 0 auto 40px;
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $text-color;
```

Comment:
what if we gonna add text with some other color to our project? The variable names should be descriptive, like `primary-gray`, or `secondary-gray`.